packages/elements/src/calendar/index.ts

```diff
@@ -1407,7 +1431,7 @@ export class Calendar extends ControlElement implements MultiValue {
         .value=${cell.value}
         .index=${cell.index}
       >
-        ${cell.text}
+        ${cellContent}
```

Comment:
cellContent is confuse me with part cell-content above. how about change it to slotContent?

Reply:
That's sound better.